setup.py

```diff
@@ -0,0 +1,48 @@
+"""Setup module"""
+
+import os
+import codecs
+from setuptools import setup, find_packages
+
+parent_dir = os.path.abspath(os.path.dirname(__file__))
+with codecs.open(os.path.join(parent_dir, 'README.md'), encoding='utf-8') as readme:
+    long_description = readme.read()
+
+setup(
+    name='monitoring-scripts',
+
+    use_scm_version=True,
+    setup_requires=['setuptools_scm'],
+
+    description='Carpe Noctem Tactical Operations Services monitoring scripts',
+    long_description=long_description,
+
+    author='Enrico Ghidoni',
```

Comment:
Make that `CNTO Research and Development branch`

Reply:
You mean author?

Reply:
yeah, Ideally we would want it to be the same as https://github.com/CntoDev/triscord/blob/master/LICENSE.txt#L1

Worth noting though that this field is only used when uploading to PyPI, so I'm not sure if it really matters as of now.

Reply:
I will anyway remove any personal reference

Reply:
As soon as I'm finished setting up an automated way of generating the AUTHORS file your name will be back in the authorship information

Reply:
But the AUTHORS file would contain the only trace, I was speaking of removing personal references from `setup.py`

Reply:
Absolutely, I was merely making sure you wouldn't feel your authorship was "stolen" in any way